proto/regen/ecocredit/marketplace/v1/events.proto

```diff
@@ -2,44 +2,27 @@ syntax = "proto3";
 
 package regen.ecocredit.marketplace.v1;
 
+option go_package = "github.com/regen-network/regen-ledger/x/ecocredit/marketplace";
+
 import "gogoproto/gogo.proto";
 import "cosmos/base/v1beta1/coin.proto";
 import "google/protobuf/timestamp.proto";
 
-option go_package = "github.com/regen-network/regen-ledger/x/ecocredit/marketplace";
-
 // EventSell is an event emitted when a sell order is created.
 message EventSell {
 
-  // order_id is the unique ID of sell order.
-  uint64 order_id = 1;
-
-  // batch_denom is the credit batch being sold.
-  string batch_denom = 2;
+  // seller is the owner of the sell order.
+  string seller = 1;
 
-  // quantity is the quantity of credits being sold.
-  string quantity = 3;
-
-  // ask_price is the price the seller is asking for each unit of the
-  // batch_denom. Each credit unit of the batch will be sold for at least the
-  // ask_price or more.
-  cosmos.base.v1beta1.Coin ask_price = 4;
-
-  // disable_auto_retire disables auto-retirement of credits which allows a
-  // buyer to disable auto-retirement in their buy order enabling them to
-  // resell the credits to another buyer.
-  bool disable_auto_retire = 5;
-
-  // expiration is an optional timestamp when the sell order expires. When the
-  // expiration time is reached, the sell order is removed from state.
-  google.protobuf.Timestamp expiration = 6 [ (gogoproto.stdtime) = true ];
+  // order_id is the unique ID of sell order.
+  uint64 order_id = 2;
 }
 
 // EventUpdateSellOrder is an event emitted when a sell order is updated.
 message EventUpdateSellOrder {
 
-  // owner is the owner of the sell orders.
-  string owner = 1;
+  // seller is the owner of the sell order.
+  string seller = 1;
```

Comment:
switched to `owner`

Reply:
> I think seller makes sense but we should probably be consistent. I would support those changes here or we can open a separate issue to do a more thorough audit of all proto fields and comments for marketplace.

Apologies again. My comment was not very clear. I think we should be using `seller` rather than `owner` for consistency but this can be handled in a followup when auditing the proto files.